Specs/DateTime/Spanish/DatePeriodParser.json

```diff
@@ -4070,11 +4070,11 @@
     "Context": {
```

Comment:
Does this also work with los anos?

Reply:
> Does this also work with los anos?

Adding support for words without accents, please review again.